### ISSUE TITLE
TextView links not handled consistently across platforms

### ISSUE THREAD

**Issue by cookieguru** [eclipsesource/tabris-js#1644]
### Problem description

- Links are not rendered consistently across all platforms:

![Android](https://user-images.githubusercontent.com/1888809/39894606-48ce6278-545c-11e8-8a81-98634a731b80.png)  ![Windows](https://user-images.githubusercontent.com/1888809/39894607-48e4e606-545c-11e8-8bf2-507d1cc6ce6b.png) ![iOS](https://user-images.githubusercontent.com/1888809/40041324-673ed0e8-57d2-11e8-8dee-e66a794616af.jpg)

- On Windows, a `tap` event bound to a TextView does not fire if the area touched is an `<a>`.  On Android, it does.  Adding a `href` attribute to the `<a>` makes no difference.

Summary of tap event firing:

|         | On `<a>` | Outside `<a>` |
|---------|----------|---------------|
| Android | ✓        | ✓             |
| iOS     | ✓        | ✓             |
| Windows | ✘        | ✓             |

### Expected behavior

- Links are rendered as links in line with the platform's native styes.  It should be easy to distinguish which parts of the text area link and which parts are not.
- Tap event fires consistently across platforms.  Ideally, a tap event handler would not be necessary: #286
- If possible, the mouse cursor changes to a hand when hovering over the cursor.  This is already the case on Windows, but not on Android.

### Workaround
On all platforms the `<ins>` tag can be used to display underlined text that is the same color as the other parts of the string.

### Environment

- Tabris.js version: 2.4.2
- OS: Android 8.0.0
- OS: iOS 11.3
- OS: Windows 10 Pro

### Code snippet

```javascript
const {TextView, ui} = require('tabris');

return new TextView({
  text: '<a>This is a link</a> and this isn\'t',
  markupEnabled: true,
}).on('tap', () => {
  console.log('I have been tapped');
}).appendTo(ui.contentView);
```

**Comment by iteearmah** [comment]
+1

**Comment by mpost** [comment]
We will add a new event `tapLink` which fires when the link is selected. The event object will contain the `href` value.

**Comment by mpost** [comment]
Implemented 2.6
